### PR TITLE
#2927 - Cannot use content templates in list views

### PR DESCRIPTION
**Pull request by KimHolzmann**
The logic is in place now.
need someone to look at
1. Styling of the buttons (dropdown)

### Prerequisites

- [ ] I have [created an issue](https://github.com/umbraco/Umbraco-CMS/issues) for the proposed changes in this PR, the link is:
- [ ] I have added steps to test this contribution in the description below

### Description

Fixes: https://github.com/umbraco/Umbraco-CMS/issues/2927

### CODE REVIEW

**Comment by KimHolzmann** [comment]
Button logic fixed now.

**Comment by KimHolzmann** [comment]
@nul800sebastiaan anyone have the time to look at this PR ?
I think it's better having a working functionality, with some styling still to do, 
than not have it at all :-)

**Comment by nul800sebastiaan** [comment]
@KimHolzmann I don't think that's how it works, ugly in this case just looked like there was something seriously wrong.. ;-)

Anyway, got it looking a bit nicer now with a few updates: 

image

**Comment by nul800sebastiaan** [comment]
Looks okay now and works as expected, thanks very much @KimHolzmann ! 👍 🎖

**Comment by KimHolzmann** [comment]
Thanks, just glad to help where I can :-)
Nice work from you to 👍

File: src/Umbraco.Web.UI.Client/src/views/propertyeditors/listview/listview.html [hunk 1]
```diff
@@ -11,35 +11,85 @@
...
+                               </a>
+                               <a ng-repeat="(key, value) in contentType.blueprints" ng-click="createFromBlueprint(entityType,contentType.alias , key)">
+                                   &nbsp;&nbsp;<i class="{{contentType.icon}}"></i>
+                                   {{value}}
+                               </a>
+                           </li>
+                       </ul>
                    </div>
+
+                   <ul class="umb-actions umb-actions-child">
```
Comment (Shazwazza) [inline src/Umbraco.Web.UI.Client/src/views/propertyeditors/listview/listview.html]: I'm unsure what this block of code is doing? As far as I can tell there is no `$scope.docType` object so I don't think this entire html block is actually used, can you clarify @KimHolzmann  ?

**Pull request merged**